### DISCUSSION TITLE
ISCE processes ERS data downloaded by ESA

### DISCUSSION THREAD

**Discussion by DJschao** [isce-framework/isce2#423] [Q&A] [answered]
Hello everyone:

The problem I currently encounter is how to use ISCE to process ERS-2 data downloaded from the official website of ESA, because the data format of ESA is .E2, and the example given in the ISCE tutorial is ERS data from WInSAR and UNAVCO. They are completely different. So I don't know how to write the InsarApp.xml file about ESA's ERS-2. 

Please help me, thank you all!

**Comment by fdelgadodelapuente** [comment, marked as answer]
Hi, in the input file use something like

	<component name="reference">
       <property name="IMAGEFILE">SAR_IMS_1PNESA19951208_211547_00000018A006_00458_03318_0000.E2 </property>
      <property name="ORBIT_TYPE">ODR</property>  
       <property name="ORBIT_DIRECTORY">/home/fdelgado/isce/esa/ers/odr/ERS2</property>
       <property name="OUTPUT">reference</property>
	</component>

	<component name="secondary">
       <property name="IMAGEFILE">SAR_IMS_1PNESA19951207_211544_00000018G151_00458_22991_0000.E1</property>
       <property name="ORBIT_TYPE">ODR</property>  
       <property name="ORBIT_DIRECTORY">/home/fdelgado/isce/esa/ers/odr/ERS1</property>
       <property name="OUTPUT">secondary</property>
	</component>

Francisco

**Reply by EJFielding** [reply]
Hello @DJschao,
Yes, the examples for ERS data processing are for the old ESA data format that is stored at WInSAR and some other archives. Newly processed ERS data has what we call "Envisat" format because it is the same as they use for Envisat data. You need to use the `ERS_EnviSAT` sensor type for the ERS raw (level 0) data in Envisat format or `ERS_EnviSAT_SLC` sensor for ERS SLC (level 1) data in Envisat format. The file name specification that @fdelgadodelapuente listed is the way to specify the input files.
++Eric

**Reply by DJschao** [reply]
Thanks a lot for your answer, currently I need to update my ISCE version (ISCE2.2.0) to handle the ERS_EnviSAT sensor type.

**Reply by EJFielding** [reply]
Yes, you need at least ISCE2 v.2.3.2 that was released more than two years ago (https://github.com/isce-framework/isce2/tree/v2.3.2) to use the `ERS_EnviSAT_SLC` sensor. You should also switch to the `stripmapApp.py` for processing instead of the obsolete `insarApp.py`.

**Reply by DJschao** [reply]
Thanks for your help, I'm trying these.
